src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -2562,3 +2698,57 @@ map<long, long> JasmineGraphFrontEnd::getOutDegreeDistributionHashMap(map<long,
     }
     return distributionHashMap;
 }
+
+void JasmineGraphFrontEnd::initiateEntityResolutionCoordinator(std::string graphID, SQLiteDBInterface sqlite, std::string masterIP, string designatedWorkerHost, string designatedWorkerPort) {
```

Comment:
This function signature is too lengthy. Lets break this into two.

Reply:
Function declaration split and adjusted